controllers/widget.js

```diff
@@ -285,13 +284,14 @@ function _onTouchend(e) {
         _applyOuterProperties(_properties.defaultStyle);
         _applyInnerProperties(_properties.defaultStyle);
     }
+}
 
-    if (e.type === 'touchend') {
```

Comment:
I used `touchend` because for the best visual result the click should be fired on release. This is done automatically for buttons, but not with views/labels.